src/modules/SdnDiag.Common/public/Start-SdnNetshTrace.ps1

```diff
@@ -31,7 +31,7 @@ function Start-SdnNetshTrace {
     param (
         [Parameter(Mandatory = $true, ParameterSetName = 'Local')]
         [Parameter(Mandatory = $true, ParameterSetName = 'Remote')]
-        [SdnRoles]$Role,
+        [String]$Role,
```

Comment:
How about add same ValidateSet as Start-SdnEtwTraceCapture for $Role?

Reply:
@luyaof good catch!